build/common.props

```diff
@@ -73,6 +73,13 @@
     </PackageReference>
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.PublicApiAnalyzers" Version="3.3.4">
```

Comment:
Do the latest preview versions have any relevant improvements? Didn't see a changelog for them.

https://www.nuget.org/packages/Microsoft.CodeAnalysis.PublicApiAnalyzers/3.11.0-beta1.24324.1

Reply:
Same, couldn't find a reason to take the preview versions.